toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -194,8 +195,8 @@ func (c *Chroot) Initialize(tarPath string, extraDirectories []string, extraMoun
 		if err != nil {
 			if buildpipeline.IsRegularBuild() {
 				// mount/unmount is only supported in regular pipeline
-				// Best effort cleanup in case mountpoint creation failed mid-way through
-				cleanupErr := c.unmountAndRemove(leaveChrootOnDisk)
+				// Best effort cleanup in case mountpoint creation failed mid-way through. We will not try again so treat as final attempt.
+				cleanupErr := c.unmountAndRemove(leaveChrootOnDisk, true)
```

Comment:
Yeah, now this feels more cryptic with just a `true`. Also, I'd rather stick to how we try to do things in the rest of the code.

Reply:
Perhaps change it to `true /*isFinalAttempt*/`?

Reply:
I think I'm going to go with:
``` golang
const (
	doLazyUnmount   = true
	doNormalUnmount = !doLazyUnmount
)
```